### ISSUE TITLE
noarch doesn't seem to create the windows entry point

### ISSUE THREAD

**Issue by moorepants** [conda-forge/skijumpdesign-feedstock#8]
Issue: On unix, after install, the user can simply type `skijumpdesign` to launch the web app, but on windows the entry point is not on the path, so the user must type the full path.

<br/>
Environment (<code>conda list</code>):
<details>

```
$ conda list

```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info

```
</details>

I installed the package on windows but the entry point is not present. I think that the skijumpdesign.exe file is not created with the noarch specification because it builds on linux.

**Comment by moorepants** [comment]
Maybe adding the entry point in the meta.yaml will work:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#python-entry-points

**Comment by moorepants** [comment]
Couple of workarounds:

```
python -m skijumpdesignapp
```

works because this module is in the path.

also

```
python C:\path\to\anaconda\Scripts\skijumpdesign
```

**Comment by moorepants** [comment]
fixed with #10
